wdlconfig.qmd

```diff
@@ -0,0 +1,58 @@
+
+# WDL Configuration Guide {{< iconify file-icons wdl >}} {#sec-wdlconfig}
+
+So as not to reinvent the wheel, WILDS WDL's should follow guidelines similar to the ones provided by [BioWDL](https://biowdl.github.io/styleGuidelines.html) and [WARP](https://broadinstitute.github.io/warp/docs/Best_practices/suggested_formats). However, because of the pedagogical "proof-of-concept" nature of the WILDS, they will not be identical and even differ significantly in a few places.
+
+## WILDS Philosophy
+
+- The mindset behind WILDS is for each repository to be a self-contained demonstration of a particular bioinformatic functionality:
+    1. Researcher scans the workflow to deem whether it is relevant to their needs.
+    2. Researcher clones the repository as is, makes minimal updates to the inputs, and easily executes the code locally or otherwise.
+    3. Researcher forks the repository and customizes it as necessary to fit their exact research needs.
```

Comment:
A little confused about these steps. Like are these what we expect the user to do? My take would be that we add a little more information. Something perhaps like this:
1. Each repository will have a Readme file that will detail the over-arching goals of that workflow (for example this workflow will help you convert a fastq file into a cram file). Provide some details about what type of file sources are expected etc. basically everything a user needs to know about the workflow in order to decide if they want to fork and run the workflow. 
2. Once you have decided that this repository is useful to you, you can clone the repository as is, specify your inputs and execute the workflow. 
3. You can also take inspiration from a repository and create a new workflow and then submit it back to the WILDS for review and inclusion.